build-docker.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-# Note: Avoid usage of arrays as MacOS users have an older version of bash (v3.x) which does not supports arrays
+# Note: Avoid usage of arrays as MacOS users have an older version of bash (v3.x) which does not support arrays
```

Comment:
FYI I think we should avoid making changes in build.sh or build-docker.sh or really any of the files from the upstream repo if we can avoid it.  Corrections are opportunities for merge conflicts when we merge with upstream changes.

This philosophy guides a lot of the changes in this repo - add new content instead of changing existing content so that as the base rpi image evolves, we can seamlessly keep up.